core/app/models/spree/payment_method/bogus_credit_card.rb

```diff
@@ -55,7 +55,7 @@ def void(_response_code, _credit_card, _options = {})
       ActiveMerchant::Billing::Response.new(true, 'Bogus Gateway: Forced success', {}, test: true, authorization: '12345')
     end
 
-    def cancel(_response_code)
+    def try_void(_payment)
```

Comment:
A YARDoc comment would be helpful here, the return values are a little confusing:
* `ActiveMerchant::Billing::Response` with `true` if the void succeeded
* `ActiveMerchant::Billing::Response` with `false` if the void failed
* Just `false` if it can't be voided at this time.

Reply:
added